libs/libqmi/Makefile

```diff
@@ -11,17 +11,22 @@ PKG_NAME:=libqmi
 PKG_VERSION:=1.30.4
 PKG_RELEASE:=$(AUTORELEASE)
 
-PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.xz
-PKG_SOURCE_URL:=https://www.freedesktop.org/software/libqmi
-PKG_HASH:=00d7da30a4f8d1185f37cba289cfaf1dfcd04a58f2f76d6acfdf5b85312d6ed6
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://gitlab.freedesktop.org/mobile-broadband/libqmi.git
+PKG_SOURCE_VERSION:=$(PKG_VERSION)
+PKG_MIRROR_HASH:=537eae29c36aba9757afd86e48b91c37c3fe3232037ad11fdd426297f6040a6b
 
 PKG_MAINTAINER:=Nicholas Smith <nicholas@nbembedded.com>
 
 PKG_INSTALL:=1
-PKG_BUILD_PARALLEL:=1
+PKG_BUILD_DEPENDS:=python3/host
```

Comment:
Why is this a required change now?

Reply:
I'm asking because I tried a build with these patches and it broke because of some failure building `python3/host`

Reply:
Meson build requires python3 https://gitlab.freedesktop.org/mobile-broadband/libqmi/-/blob/main/meson.build#L48
It worked for me. I will check it again.

Reply:
> I'm asking because I tried a build with these patches and it broke because of some failure building `python3/host`

All build checks have just passed successfully. No problem.

Reply:
glib2 requires meson and it doesn't depend on `python3/host`. @neheb is there any rationale here to require this?

Reply:
@Linaro1985 could you post a new PR removing the explicit `python3/host` dep?